docs/examples/workflows/coinflip.md

```diff
@@ -72,12 +72,9 @@
 
             import random
 
+            result = ''heads'' if random.randint(0, 1) == 0 else ''tails''
```

Comment:
The double single quotes makes me nervous, does this run in python?

Reply:
It's yaml first, if you convert it to json, the double single quotes go away.